integ-test/src/test/java/org/opensearch/sql/sql/GeoPointIT.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.sql.sql;
+
+import org.json.JSONObject;
+import org.junit.Test;
+import org.opensearch.sql.legacy.SQLIntegTestCase;
+
+import java.io.IOException;
+import java.util.Map;
+
+import static org.opensearch.sql.legacy.TestsConstants.TEST_INDEX_GEOPOINT;
+import static org.opensearch.sql.util.MatcherUtils.rows;
+import static org.opensearch.sql.util.MatcherUtils.verifyDataRows;
+
+public class GeoPointIT extends SQLIntegTestCase {
+  @Override
+  protected void init() throws Exception {
+    loadIndex(Index.GEOPOINT);
+  }
+
+  @Test
+  public void test_geo_point() throws IOException {
+    String query = "SELECT geo_point_object FROM " + TEST_INDEX_GEOPOINT;
+    JSONObject result = executeJdbcRequest(query);
+    verifyDataRows(result,
+        rows(new JSONObject(Map.of(
+                "lat", 40.71,
+                "lon", 74))),
+        rows(new JSONObject(Map.of(
+            "lat", -33.85253637358241,
+            "lon", 151.21652352950258))),
+        rows(JSONObject.NULL)
+    );
+  }
+
+  @Test
+  public void test_geo_point_unsupported_format() throws IOException {
```

Comment:
Consider adding a test for `lat` issue and separate test for `lon` issue from `opensearch/src/main/java/org/opensearch/sql/opensearch/data/utils/OpenSearchJsonContent.java`.

Reply:
We actually cant load non-number values in to lat and lon to OpenSearch. So integtests won't work. I believe this is just a fail safe for the valueFactory. There are unit tests for this I think that would be enough. Thoughts?